### PR TITLE
Add Video in SVG Blobs on Primary and Sub Series Headers

### PR DESCRIPTION
**Pull request by ashleyjelks**
Kapture 2019-08-19 at 13 37 23

Kapture 2019-08-19 at 13 44 32

[Links to GROW-1426](https://artsyproduct.atlassian.net/browse/GROW-1426)
[Links to GROW-1427](https://artsyproduct.atlassian.net/browse/GROW-1427)

### CODE REVIEW

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/Introduction.tsx [hunk 1]
```diff
@@ -48,8 +52,18 @@ export const VanguardIntroduction: React.SFC<{
...
+  position: relative;
+  padding-bottom: 100px;
+  min-height: 100vh;
+`
+
 const HeaderText = styled(Sans)`
   font-size: 100px;
+  position: absolute;
+  top: 3vw;
+  left: 35vw;
```
Comment (eessex) [inline src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/Introduction.tsx]: We probably don't want to set the `left` value hard coded, have you tried making the width 100% and centering?

File: src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx [hunk 2]
```diff
@@ -24,7 +37,7 @@ export const VanguardSeriesWrapper: React.SFC<{
         // prevents overlapping nav on jump-link
       />
       <Box height="95vh" mb={80}>
-        <VanguardVideoBackground article={article} />
+        {isVideo && <IntroSVGVideo id={getVideoID(slugifiedTitle)} url={url} />}
         <Box mx="auto" maxWidth={1400} px={4}>
```
Comment (eessex) [inline src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx]: Based on the screenshots it looks like we may need to change the `z-index` on these titles, which appear to be behind the videos.

Comment (ashleyjelks) [reply src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx]: Yeah, the video comp itself as a `z-index` of `-1` so it _should_ be rendering behind the text but the header text and the svgs have a `position:absolute` prop which may be why this appears to be happening. Anyways I'll address in the next PR which is to get the text-inversions and transforms working!

Comment (eessex) [reply src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx]: We probably need to wrap the parent container in `position: relative`

Comment (eessex) [reply src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx]: Happy to pair on this if you want

**Comment by artsyit** [comment]
:rocket: PR was released in v17.1.6 :rocket:

**Pull request merged**